packages/data/addon/adapters/bard-facts.js

```diff
@@ -16,6 +16,30 @@ import { configHost } from '../utils/adapter';
 
 const SORT_DIRECTIONS = ['desc', 'asc'];
 
+/**
+ * Serializes a list of filters to a fili filter string
+ * @param {Array<Filter>} filters - list of filters to be ANDed together for fili
+ * @param {Filter} filter - filters object
+ * @param {String} filter.dimension - dimension to be filtered on
+ * @param {String} filter.field - the dimension field to filter
+ * @param {String} filter.operator - the type of filter operator
+ * @param {Array<String|number>} filter.values - the values to pass to the operator
+ */
+export function serializeFilters(filters) {
+  return filters
+    .map(filter => {
+      const { dimension, field, operator, values } = filter;
+      const serializedValues = array(values)
+        .toArray()
```

Comment:
Why do we need to go from wrapping the array in an `Ember.Array` then convert into a native array?